app/components/custom-smart-banner.js

```diff
@@ -0,0 +1,86 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+import RespondsToScroll from 'ember-responds-to/mixins/responds-to-scroll';
+import { system } from '../utils/browser';
+import { trackActions } from '../utils/track';
+
+export default Component.extend(RespondsToScroll, {
+  i18n: service(),
+  smartBanner: service(),
+  wikiVariables: service(),
+
+  classNames: ['fandom-app-smart-banner'],
+  dayInMiliseconds: 86400000,
+  // sync with scss variable $fandom-app-smart-banner-height
+  fandomAppSmartBannerHeight: 85,
+  trackCategory: 'custom-smart-banner',
+
+  closeButtonSelector: '.fandom-app-smart-banner__close',
+
+  text: computed('wikiVariables.smartBannerAdConfiguration.text', function () {
```

Comment:
can't we use `oneWay` here?

Reply:
I would also create another computed property which stores `wikiVariables.smartBannerAdConfiguration`. Not to repeat `wikiVariables.smartBannerAdConfiguration` for all the properties